packages/notebook-extension/schema/tracker.json

```diff
@@ -878,6 +878,12 @@
       "type": "number",
       "default": 50
     },
+    "showEditorForReadOnlyMarkdown": {
+      "title": "Show editor for read-only Markdown cells",
+      "description": "Should an editor be shown for read-only markdown",
+      "type": "boolean",
+      "default": true
+    },
```

Comment:
> This is more for discussion with core devs and don't need to be addressed by this PR.

It feels weird to have sections like `MarkdownCellConfig`, `CodeCellConfig`,... but to get `maxNumberOutputs` and `showEditorForReadOnlyMarkdown` outside those sections (aka because these `CellConfig` are actually `Cell`**Editor**`Config`). It would be good if the settings and code structure reflect that better.